lib/FSEngine/FSBSA.cpp

```diff
@@ -31,7 +31,11 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ***** END LICENCE BLOCK *****/
 
 #include "FSBSA.h"
+#ifdef __linux__
+#include <directx/dxgiformat.h>
+#elif
```

Comment:
The #elif with a missing condition is causing a compile error on MSVC. Can you change it to an else?